docker-compose.yml

```diff
@@ -129,20 +116,22 @@ services:
     volumes:
       - icingaweb:/data
 
-  icingaweb-mysql:
+  mysql:
+    # see https://hub.docker.com/_/mariadb
+    image: mariadb
     command: --default-authentication-plugin=mysql_native_password
     environment:
       MYSQL_RANDOM_ROOT_PASSWORD: 1
-      MYSQL_DATABASE: icingaweb
-      MYSQL_USER: icingaweb
-      MYSQL_PASSWORD: ${ICINGAWEB_MYSQL_PASSWORD:-icingaweb}
-    image: mysql
+      ICINGAWEB_MYSQL_PASSWORD: ${ICINGAWEB_MYSQL_PASSWORD}
+      ICINGADB_MYSQL_PASSWORD: ${ICINGADB_MYSQL_PASSWORD}
     logging: *default-logging
     volumes:
-      - icingaweb-mysql:/var/lib/mysql
+      # - ./mysql:/var/lib/mysql
+      - mysql:/var/lib/mysql
+      - ./init-mysql.sh:/docker-entrypoint-initdb.d/init-mysql.sh
 
 volumes:
   icinga2:
-  icingadb-mysql:
+  mysql:
   icingaweb:
   icingaweb-mysql:
```

Comment:
This volume could be removed as well, right? Also, I like things ordered 😆  So please put `mysql` last here.

Reply:
- [x] done